app/controllers/users_controller.rb

```diff
@@ -344,13 +362,29 @@ def update
 	end
 
 	def favorite_recipe
-		current_user.recipes.create(:type => "FavoritedRecipe", :meal_type => params[:Type], :title => params[:Title], 
-			:calories => params[:Calories], :time => params[:PrepTime]
-			)
+		favorited_recipe = Recipe.new do |r|
```

Comment:
Just wondering - why is this functionality in the users controller and not the recipes controller?

Reply:
yeah userscontroller is a bit bloated now - we can work on refactoring it in another pr.